tests/test_file.py

```diff
@@ -45,6 +45,18 @@ def test_locale(self, mozwebqa):
             print "There is no language selector on the page"
 
     @pytest.mark.nondestructive
+    def test_locale_zh_TW (self, mozwebqa):
+        main_page = MySiteHomePage(mozwebqa)
+        zh_TW_locale = ['/zh-TW']
+        for path in zh_TW_locale:
+            url = main_page.base_url + path
```

Comment:
This could have issues if the components are not separated by a single slash. Try using [https://docs.python.org/2/library/urlparse.html#urlparse.urljoin urlparse.urljoin()] to create the new URL.